scripts/cdist

```diff
@@ -20,6 +21,34 @@
 #
 #
 
+def inspect_ssh_mux_opts():
+    import subprocess
+
+    """Inspect whether or not ssh supports multiplexing options"""
+    wanted_mux_opts = {
+        "ControlPath":"~/.ssh/master-%l-%r@%h:%p",
```

Comment:
I think the control path should be relative to the temporary cdist directory so the socket vanishes when the run is over